### ISSUE TITLE
[Enhancement]: Make Series editable / contain their own metadata / make them smarter

### ISSUE THREAD

**Issue by kuldan5853** [advplyr/audiobookshelf#2946]
### Describe the feature/enhancement

Right now, Series are a very limited feature, basically only able to group books. 

I would like a way to edit a Series directly - especially changing the name. If you want to rename a Series (e.g. because the metadata automatch pulled a phrasing you didn't like), the only chance you have right now is to create a new series, losing all the sequence numbers in the process. 

I propose a way to edit a "Series" on the Series level, being able to change it's name, and also to be able to add Metadata to the Series itself (Description, a "Series" cover, mainly). 

In addition, I would suggest to split the "sequence" and the "series" when automatching from metadata - this way, you could match the sequence number from the online source, but use a different series name if so desired. 

Also, the cherry on top would be a way to nest series within each other - I know that sub-series functionality exists, but it lists the "top level series" and the "sub-series" both in library view - it would be nice if there would be a toggle to hide sub-series from the library view, for a cleaner structure.

**Comment by nichwall** [comment]
Sub series 
https://github.com/advplyr/audiobookshelf/issues/870
Covers (specifically calls out collections, but same thing)
https://github.com/advplyr/audiobookshelf/issues/2589
Series names
https://github.com/advplyr/audiobookshelf/issues/1604

**Comment by kuldan5853** [comment]
Thank you - I searched before posting, but it seems my github-fu is lacking.

**Comment by chyron8472** [comment]
I'm actually current using series covers by inserting a sort of placeholder book. I added books that are near-zero-length to my Library, then added each book to the start of a given series (as series num 0). Then I added the desired cover to said book.

It occurs to me that I could also put series-related metadata in said book entry if I wanted, which could apply as an approach to how this feature might be implemented.
